### PR TITLE
Feature: Add ETag header on output

### PR DESCRIPTION
**Pull request by n0vad3v**
This branch will add E-Tag header on images rendered with WebP Server Go.

E-Tag is calculated with the file's file-size and it's crc32 checksum, an example output will look like this: `ETag: W/"524016-39D658B9"`.

This might be useful while working with proxy or CDN in front of it.

### CODE REVIEW

File: webp-server.go [hunk 1]
```diff
@@ -141,7 +141,7 @@ func main() {
 	// Server Info
 	log.Infof("WebP Server %s %s", version, ListenAddress)
 
-	app.Get("/*", Convert(confImgPath, ExhaustPath, AllowedTypes, QUALITY))
+	app.All("/*", Convert(confImgPath, ExhaustPath, AllowedTypes, QUALITY))
```
Comment (BennyThink) [inline webp-server.go:144]: 🤔why allow all request methods?

File: helper.go [hunk 2]
```diff
@@ -73,4 +86,5 @@ func isSafari(UA string) bool {
 		return true
 	}
 	return false
+
```
Comment (BennyThink) [inline helper.go:89]: remove this line and above

File: helper_test.go [hunk 3]
```diff
@@ -58,6 +58,16 @@ func TestGenWebpAbs(t *testing.T) {
 	}
 }
 
+func TestGenEtag(t *testing.T) {
+	var data = "./pics/png.jpg"
+	var expected = "W/\"1020764-262C0329\""
+	var result = GenEtag(data)
+
+	if result != expected {
```
Comment (BennyThink) [inline helper_test.go:66]: use `assert.NotEqual`, document for [assert](https://godoc.org/github.com/stretchr/testify/assert)

File: webp-server.go (hunk 1 above)
Comment (n0vad3v) [reply webp-server.go:144]: Used for catch all.

**Pull request merged**